src/aggregate_loader.py

```diff
@@ -1,178 +1,217 @@
-import logging, coloredlogs, os, multiprocessing, time, argparse, time
-
-from etl import *
-from etl import VariationETL
-from etl import ConstructETL
-from etl import VEPETL
-from etl import TranscriptETL
-from etl import SequenceTargetingReagentETL
-from etl import ECOMAPETL
-from etl.helpers import Neo4jHelper
+#!/usr/bin/env python
+'''This is the main entry-point for running the ETL pipeline'''
+
+import logging
+import os
+import multiprocessing
+import time
+import argparse
+import coloredlogs
+
+from etl import ETL, MIETL, DOETL, BGIETL, ConstructETL, ExpressionAtlasETL, GenericOntologyETL, \
+                ECOMAPETL, AlleleETL, VariationETL, SequenceTargetingReagentETL, \
+                AffectedGenomicModelETL, TranscriptETL, GOETL, ExpressionETL, ExpressionRibbonETL, \
+                ExpressionRibbonOtherETL, DiseaseETL, PhenoTypeETL, OrthologyETL, ClosureETL, \
+                GOAnnotETL, GeoXrefETL, GeneDiseaseOrthoETL, MolecularInteractionETL, \
+                GeneDescriptionsETL, VEPETL, VEPTranscriptETL, Neo4jHelper
+
 from transactors import Neo4jTransactor, FileTransactor
 from data_manager import DataFileManager
 from common import ContextInfo  # Must be the last timeport othersize program fails
 
 
-parser = argparse.ArgumentParser(description='Load data into the Neo4j database for the Alliance of Genome Resources.')
-parser.add_argument('-c', '--config', help='Specify the filename of the YAML config. It must reside in the src/config/ directory', default='default.yml')
-parser.add_argument('-v', '--verbose', help='Enable DEBUG mode for logging.', action='store_true')
-args = parser.parse_args()
-
-# set context info
-context_info = ContextInfo()
-context_info.config_file_location = os.path.abspath('src/config/' + args.config)
-if args.verbose:
-    context_info.env["DEBUG"] = True
-
-debug_level = logging.DEBUG if context_info.env["DEBUG"] else logging.INFO
-
-coloredlogs.install(level=debug_level,
-                    fmt='%(asctime)s %(levelname)s: %(name)s:%(lineno)d: %(message)s',
-                    field_styles={
+def main():
```

Comment:
I'm not sure this change, though it does conform to the long line requirements, makes the code more readable.  In particular, line 30.

Reply:
Who these days still uses a 80 char wide monitor? or 160 for that matter? I get it if you need to look at this on a laptop then you can't see everything and it line wrapped but from a strictly readability the more its all on one line the better.

Reply:
This is clearly mainly a preference thing. But what Pylint / PEP8 uses defaults with is line length 120. This I especially find useful for reading code in GitHub. I, in general, don't think line length is a big deal but prefer to have lines be broken up for the sake of readability.

Reply:
Well it is just that, preference... I find it much easier when everything is on one line. You don't have to go searching with your eyes for the rest of the line, or have to deal with python formatting when it's all on one line. So with this commit we are taking your preference above all others.